packages/manager/modules/octavia-load-balancer/src/onboarding/controller.js

```diff
@@ -25,4 +34,13 @@ export default class OctaviaLoadBalancerOnboardingCtrl {
       type: 'navigation',
     });
   }
+
+  createLoadBalancer($event) {
+    if (this.hasPrivateNetwork) {
+      this.goToLoadBalancerCreation();
+    } else {
+      this.goToNoPrivateNetwork();
+      $event.preventDefault();
```

Comment:
**question:** Why do you need this?

Reply:
I need this preventDefault() because the click event is listening on a link (<a>), so when you click the link, if there are no private network, we show the popup and preventDefault because, if not, the link is followed. So this way, we prevent navigation and stay on the page to display the popup.